libs/ledgerjs/packages/hw-app-aptos/src/Aptos.ts

```diff
@@ -15,10 +15,10 @@
  *  limitations under the License.
  ********************************************************************************/
 
-import BIPPath from "bip32-path";
 import { sha3_256 as sha3Hash } from "@noble/hashes/sha3";
 import Transport from "@ledgerhq/hw-transport";
 import { StatusCodes } from "@ledgerhq/errors";
+import { bip32asBuffer } from "@ledgerhq/coin-framework/bridge/jsHelpers";
```

Comment:
you need to import it from bip32 lib, not coinframework, we don't want this dependency